### ISSUE TITLE
Clone Extension

### ISSUE THREAD

**Issue by echoless3484** [TurboWarp/extensions#333]
Extension to Export all clones, where they are and what effects they have, all the clones and their data into a list, than another block to import all clones in that list into that project, placing them with their code, basically an easy saving and loading system for clones.

**Comment by GarboMuffin** [comment]
I don't see saving a big list of clones as a viable approach for game state saving for various reasons

**Comment by echoless3484** [comment]
Yeah, this is an all-around terrible approach. I've learned a lot in the past few years.
